package.json

```diff
@@ -20,16 +20,17 @@
     "@babel/plugin-proposal-object-rest-spread": "^7.6.2",
     "@babel/preset-env": "^7.6.3",
     "@babel/preset-react": "^7.6.3",
+    "alex": ">=8.0.1",
     "babel-eslint": "^10.0.1",
     "babel-loader": "^8.0.5",
+    "copy-webpack-plugin": "^5.1.1",
```

Comment:
Why are the files modified instead of deleted in this PR if this is going to copy and overwrite them each time webpack is run?

Reply:
@driusan that makes sense. They're modified here because current LORIS doesn't use the same react libraries served to the frontend in htdocs that node_modules has from package.json. I can remove them because when make will generate them.